spirv_glsl.cpp

```diff
@@ -237,6 +237,21 @@ void CompilerGLSL::remap_pls_variables()
 	}
 }
 
+void CompilerGLSL::rename_interface_variable(const vector<Resource> &resources, const InterfaceVariableRename &rename)

```

Comment:
This logic belongs in main.cpp, it's a bit too specific to be part of spirv_glsl.cpp